docs/docs/get_started/deploy_to_kubernetes.md

```diff
@@ -56,23 +58,23 @@ To run through this tutorial, you will need:
 + [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) configured to point to the cluster
 + [Docker CLI](https://docs.docker.com/install/)
 
-Our Kubernetes deployment manifests assume that you are using minikube, so we use `NodePort` to expose the services; you may prefer to use a `LoadBalancer` for a GKE cluster.
+Our Kubernetes deployment manifests assume that you are using minikube, so we use **NodePort** to expose the services; you may prefer to use a **LoadBalancer** for a GKE cluster.
 
 ## Sample Application
 
 The tutorial uses an existing [pet store demo application](https://github.com/conjurdemos/pet-store-demo) that exposes the following routes:
 
 - `GET /pets` to list all the pets in inventory
 - `POST /pet` to add a pet
-  - Requires `Content-Type: application/json` header and body that includes `name` data
+  - Requires **Content-Type: application/json** header and body that includes **name** data
 
 There are additional routes that are also available, but these are the two that we will focus on for this tutorial.
 
-Pet data is stored in a PostgreSQL database, and the application may be configured to connect to the database by setting the `DB_URL` environment variable in the application's environment (following [12-factor principles](https://12factor.net/)).
+Pet data is stored in a PostgreSQL database, and the application may be configured to connect to the database by setting the `$DB_URL` environment variable in the application's environment (following [12-factor principles](https://12factor.net/)).
 
 See [Try The Running Application](#try-the-running-application) for examples of consuming the routes using `curl`.
 
-The database is **credential-protected**. With the Secretless Broker, we will be able to use a value of `DB_URL` of the form:  `postgresql://x@localhost:5432/${APPLICATION_DB_NAME}?sslmode=disable`. The application will not require knowledge of credentials to connect to the database.
+The database is **credential-protected**. With the Secretless Broker, we will be able to use a value of `$DB_URL` of the form:  `postgresql://localhost:5432/${APPLICATION_DB_NAME}?sslmode=disable`. The application will not require knowledge of credentials to connect to the database.
```

Comment:
s/"`$DB_URL`"/"`DB_URL` environment variable". Ditto for a few other examples of this in this file.

Reply:
addressed